wagtail/admin/templates/wagtailadmin/pages/action_menu/menu_item.html

```diff
@@ -0,0 +1,7 @@
+<li>
+    {% if url %}
+        <a href="{{ url }}">{{ label }}</a>
+    {% else %}
+        <input type="submit" name="{{ name }}" value="{{ label }}" class="button" />
```

Comment:
Any reason why ``<input type="submit">`` is used here rather than ``<button type="submit">`` that is used elsewhere?